style.css

```diff
@@ -74,6 +74,200 @@ nav a:hover {
   color: #505f79;
 }
 
+.icons li a {
+  color: #505f79;
+}
+
 .headline li a:hover {
   cursor: pointer;
 }
+
+.portfolio {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  margin: 70px 3.2%;
+}
+
```

Comment:
- [ ] The background color of `work` section in the template is `#E5E5E5` and according to the requirement we should use exact same colors from template. Kindly change the background color of work section to  `#E5E5E5`. 

| Figma design | Your design |
|--------------|--------------|
|<img width="300" alt="Screenshot 2023-04-29 at 1 55 23 PM" src="https://user-images.githubusercontent.com/44798044/235294738-dc3e3b63-f3c9-485f-bde9-76e19a1c2595.png">|<img width="369" alt="Screenshot 2023-04-29 at 1 55 51 PM" src="https://user-images.githubusercontent.com/44798044/235294749-dd6b4ca6-44d9-4c14-b6c7-e4c86da4a7ef.png">|

Reply:
I done this task by adding background color as required.